README.md

```diff
@@ -9,16 +9,12 @@ var ns = require('mongodb-ns');
 
 var bacon = ns('canadian-things.songs-aboot-bacon');
 console.log(bacon.toString(), bacon);
-
-// knows how to handle graphite-style metrics to boot
-var maple = ns('canadian-things.maple-based-fabrics.read.time');
-console.log(maple.toString(), maple);
 ```
 
 will output
 
 ```
-canadian-things.songs-aboot-bacon {
+canadian-things.songs-aboot-bacon NS {
```

Comment:
Good find, made clearer in https://github.com/mongodb-js/ns/commit/f82bf983307f8774fbe64c5ec192c2fc0206734d